FLEDGE.md

```diff
@@ -527,15 +527,25 @@ For the JSON response, only the `https` scheme is supported -- the `uuid-in-pack
 
 #### 2.5.2 Using Response Headers
 
-An alternative way to pass DirectFromSellerSignals without subresource bundles is via the `Ad-Auction-Signals` response header of some `fetch()` request, together with the `directFromSellerSignalsHeaderAdSlot` parameter on `navigator.runAdAuction()`.
+An alternative way to pass DirectFromSellerSignals without subresource bundles is via the `Ad-Auction-Signals` response header of some `fetch()` request or `iframe` navigation, together with the `directFromSellerSignalsHeaderAdSlot` parameter on `navigator.runAdAuction()`.
 
-With this method, a [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) call is made by some script on the page (this call may be made in a subframe), with an extra option, `{adAuctionHeaders: true}`:
+To pass DirectFromSellerSignals using a [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) call made by some script on the page (including in a subframe), specify an extra option, `{adAuctionHeaders: true}`:
 
 ```javascript
 let fetchResponse = await fetch("https://seller.com/signals", {adAuctionHeaders: true});
 ```
 
-The browser will make the request it would without `{adAuctionHeaders: true}`, with the exception that the request will also include a request header, `Sec-Ad-Auction-Fetch: ?1`. This header indicates to the server that any `Ad-Auction-Signals` response header from the server will only be loaded in auctions via `directFromSellerSignalsHeaderAdSlot` (this is analogous to the guarantees of `Ad-Auction-Only` and `Sec-Fetch-Dest: webbundle` from the [subresource bundle version](#251-using-subresource-bundles) -- scripts on the page cannot set the `Sec-Ad-Auction-Fetch: ?1` request header without using the `{adAuctionHeaders: true}` option).
+The script must resolve the `directFromSellerSignalsHeaderAdSlot` Promise only after the response for this call is received. If the script chooses to call `runAdAuction()` after this response is received, the Promise may be immediately resolved.
+
+To pass DirectFromSellerSignals using an [`iframe`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) navigation, specify the `adAuctionHeaders` attribute on the `iframe` element:
+
+```html
+<iframe src="..." adAuctionHeaders></iframe>
+```
+
+The script that invokes `runAdAuction()` will be part of the response from that iframe navigation, and so because `runAdAuction()` is not called until after the response is received, the `directFromSellerSignalsHeaderAdSlot` Promise may be immediately resolved.
```

Comment:
```suggestion
If script that invokes `runAdAuction()` is part of the response to that iframe navigation, the `directFromSellerSignalsHeaderAdSlot` Promise may be immediately resolved from within the iframe because `runAdAuction()` cannot be called until after the response is received, .
```

Reply:
actually instead of saying "Promise may be immediately resolved", do we instead want to say `directFromSellerSignalsHeaderAdSlot` can be specified directly without a Promise.

Reply:
Good call. I applied both changes. Thanks!